test/performance/kueue/job.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.Name}}
+  labels:
+    group: test-job
+  annotations:
+    kueue.x-k8s.io/queue-name: {{.LocalQueue}}  
```

Comment:
If possible, share the file with the other configuration. If kueue is not installed, this annotation is ignored, so it's not bad to have it in both configurations.

Reply:
We still need to remove this annotation if Kueue IS installed, but we still want to test regular Kubernetes Job API performance. If not, Kueue suspends the Job and complains that the local queue is not there. So, I added:
```  
{{if .UseKueue}}
  annotations:
    kueue.x-k8s.io/queue-name: {{.LocalQueue}}
{{end}}
```